chapter-08/end/States/Dev/DevState.cs

```diff
@@ -0,0 +1,74 @@
+﻿using chapter_08.Engine.Input;
+using chapter_08.Engine.States;
+using chapter_08.Input;
+using chapter_08.Objects;
+using chapter_08.States.Particles;
+using Microsoft.Xna.Framework;
+
+namespace chapter_08.States
+{
+    public class DevState : BaseGameState
+    {
+        private const string ExhaustTexture = "Cloud";
+        private const string MissileTexture = "Missile";
+        private const string PlayerFighter = "fighter";
+
+        private ExhaustEmitter _exhaustEmitter;
+        private MissileSprite _missle;
+        private PlayerSprite _player;
+
+        public override void LoadContent()
+        {
+            var exhaustPosition = new Vector2(_viewportWidth / 2, _viewportHeight / 2);
+            //_exhaustEmitter = new ExhaustEmitter(LoadTexture(ExhaustTexture), exhaustPosition);
+            //AddGameObject(_exhaustEmitter);
```

Comment:
Intentionally commented out?

Reply:
yeah. I was playing with the emitter on its own before adding it to the missile. It's useful to uncomment it to play with the particle attributes when needed.